msteams-platform/feedback.md

```diff
@@ -1,29 +1,36 @@
+|        **Tab SDK issues/suggestions**        |  Submit issues and feature requests to the SDK repository for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js/issues). Use the `microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams).                                                                                                                                                                                                                       |
+|            **Bot & Messaging Extension SDK issues/suggestions**             |      Submit issues and feature requests to the SDK repository for your bot's language ([C#](https://github.com/Microsoft/botbuilder-dotnet/), [Javascript](https://github.com/Microsoft/botbuilder-js), or [Python](https://github.com/Microsoft/botbuilder-python)). How-to questions about the SDK can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework%20microsoft-teams) using the `botframework & microsoft-teams` tag.                                                                                            |
+| **Samples** |             Submit issues with samples to respective [Microsoft Teams Samples](/microsoftteams/platform/tutorials/code-samples) repository.                                                                                                                                                                                            |
+| **Features suggestions**             |      Make your suggestions for a feature on the [Microsoft Teams feedback portal on UserVoice](https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform).                                                                                                                                                            |
```

Comment:
You can post features suggestions on (https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform.